recipes/pyroSAR/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "pyroSAR" %}
+{% set version = "0.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1008a6ed23bc3509b511af634360bccfb187a81745cdc5a1760c8ad3aa47bbe4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools_scm
+  run:
+    - python
+    - setuptools_scm
+    - gdal >=2.2
+    - spatialist ==0.5
```

Comment:
Can this be looser, like >=0.5

Reply:
Sure. I develop `spatialist` and `pyroSAR` in parallel and requiring a fixed version was easier to maintain. I agree though that this is not optimal.